lib/heex_formatter/core/eex_formatter.ex

```diff
@@ -0,0 +1,66 @@
+defmodule HeexFormatter.Core.EexFormatter do
+  @moduledoc """
+  Format eex syntax.
+  """
+  @tab HeexFormatter.Phases.Format.tab()
+  def format(code, opts \\ []) do
```

Comment:
I would keep this code in the `Format` module for now and see how it scale. Most of the functions would be private + some docs to help what is going on.

After we see if it makes sense to break it down and how. Does it make sense?